resources/views/dashboard/index.blade.php

```diff
@@ -2,29 +2,64 @@
 
 @section('body_class', 'body__home')
 
-@section('content')
-
-    <call-to-action></call-to-action>
+@section('styles')
+<link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/@splidejs/splide@latest/dist/css/splide.min.css">
+<script src="https://cdn.jsdelivr.net/npm/@splidejs/splide@latest/dist/js/splide.min.js"></script>
+@endsection
 
-    <div class="uppercase text-center mb-2 mt-2 whitespace-pre-line">
-        <h1>Karls Advanced Remote Loadout</h1>
-        <h2 class="pr-4 pl-4">lets you build custom deep rock galactic loadouts to share with your fellow employees.
-            Get started by selecting a class above or choose a popular loadout below.</h2>
+@section('content')
+    <h1 class="ml-1">Find Loadouts by Class:</h2>
+    <div class="flex flex-row mb-5">
+        <div class="flex flex-col w-full">
+            <div class="flex flex-row">
+                <a href="/browse?character=3" class="flex w-1/4 px-4 py-8 dashImageD dashD mx-1 align-middle text-xl">
+                        Driller
+                </a>
+                <a href="/browse?character=1" class="flex w-1/4 px-4 py-8 dashImageE dashE mx-1 align-middle text-xl">
+                    Engineer
+                </a>
+                <a href="/browse?character=4" class="flex w-1/4 px-4 py-8 dashImageG dashG mx-1 align-middle text-xl">
+                    Gunner
+                </a>
+                <a href="/browse?character=2" class="flex w-1/4 px-4 py-8 dashImageS dashS mx-1 align-middle text-xl">
+                    Scout
+                </a>
+            </div>
+        </div>
+        
+        
     </div>
-    <x-dashboard-listing :loadoutList="$recentTopLoadouts" title="Top Loadouts in Past 2 Weeks" />
-    <x-dashboard-listing :loadoutList="$latestLoadouts" title='Newest Loadouts' />
-
+    
     <div class="featuredLoadoutsContainer">
-        <h1 class="uppercase text-center">Most Popular Loadouts -- All Time</h1>
-        <div class="cardGroups flex flex-wrap mb-4 -mx-1">
-            @foreach($allTimeTopLoadouts as $characterLoadouts)
-            <div class="loadoutCards w-full lg:w-1/2 px-1">
-                @foreach($characterLoadouts as $loadout)
-                <x-dashboard-loadout :loadout="$loadout"/>
-                @endforeach
+        <div class="flex flex-row">
+            <div class="flex flex-col w-1/2">
+                <x-dashboard-listing :loadoutList="$recentTopLoadouts" title="Top Loadouts in Past 2 Weeks" />
+            </div>
+            <div class="flex flex-col w-1/2">
+                <x-dashboard-listing :loadoutList="$latestLoadouts" title='Newest Loadouts' />
             </div>
-            @endforeach
         </div>
     </div>
 
+    <div class="featuredLoadoutsContainer px-3 pb-2">
+        @foreach ($latestPosts as $post)
+            <div class="my-5">
+                <a href="/blog/{{$post->id}}" class="hover:underline"><h3>{{$post->title}}</h3></a>
```

Comment:
Should use the `route` function here.

Something like `href="{{ route('blog.show', $post->id) }}"`

Reply:
Bada bing